share/ncs-package/cmake/NcsConfig.cmake

```diff
@@ -60,3 +60,8 @@ if(NOT NO_BOILERPLATE)
 
   include(${NRF_DIR}/cmake/boilerplate.cmake NO_POLICY_SCOPE)
 endif()
+
+if("${CMAKE_FIND_PACKAGE_NAME}" STREQUAL Ncs)
+  # nRF Connect SDK has been included using `find_package(NCS)` so let's find Zephyr.
+  find_package(Zephyr REQUIRED)
```

Comment:
Does this mean that including the Zephyr through the NCS package would allow any version of Zephyr to be included ?

Reply:
> Does this mean that including the Zephyr through the NCS package would allow any version of Zephyr to be included ?

No, the Zephyr being selected here would be the Zephyr residing in the same workspace as the NCS.
And that Zephyr version is controlled by the manifest file:
https://github.com/nrfconnect/sdk-nrf/blob/8bbe3ca588bf7feff431492923af3633b549ddd0/west.yml#L55-L57